pkg/router/template/types.go

```diff
@@ -62,6 +62,7 @@ type Endpoint struct {
 	Port       string
 	TargetName string
 	PortName   string
+	IDhash     string
```

Comment:
`IdHash` for consistency with the other variable names. And that affects the prior comments re: using `IDHash` for the server id.